### DISCUSSION TITLE
Wagtail tutorials video

### DISCUSSION THREAD

**Discussion by paragliding-nsk** [wagtail/wagtail#10838] [General]
Hello, I have one more "newbie" question: what courses are the best for learning Wagtail 5? I found this one:
https://www.youtube.com/playlist?list=PLMQHMcNi6ocsS8Bfnuy_IDgJ4bHRRrvub
but it's quite old and I'm not sure - is it still actual?

And of course, I know about this:
 https://docs.wagtail.org/en/stable/getting_started/index.html
But sometimes it is better to watch some compact video example instead of reading long instructions.

**Comment by RealOrangeOne** [comment]
"best" is highly subjective sadly, but there is definitely a lot of content out there. There are lots of [3rd-party guides](https://docs.wagtail.org/en/stable/advanced_topics/third_party_tutorials.html#third-party-tutorials) on the docs. The absolute core of Wagtail (ie page models, routing, fields etc) hasn't changed in a while, so any semi-recent guide ought to be fine. From there, it's choosing which bits you want to learn and going from there - the docs can be very helpful with that part.

**Reply by paragliding-nsk** [reply]
Thanks a lot!
